riotfile.py

```diff
@@ -129,7 +132,7 @@ def select_pys(min_version=MIN_PYTHON_VERSION, max_version=MAX_PYTHON_VERSION):
         Venv(
             name="appsec",
             pys=select_pys(),
-            command="pytest {cmdargs} tests/appsec",
+            command="pytest --ddtrace {cmdargs} tests/appsec",
```

Comment:
should we just add `--ddtrace` to the pytest config in `setup.cfg` ?

Reply:
I see below we are not using it for one scenario, are we able to selectively disable for a single run vs needing to enable everywhere?

Reply:
Yes, there are a few cases where it doesn't work (mainly pytest tests and similar): pytest, ci_visibility, pytest-bdd and asynctest

I can explore the option to create a `--no-ddtrace` option and see if it works.

Is it OK if I do that in a follow-up PR?